src/App.js

```diff
@@ -1,10 +1,28 @@
 import React from 'react';
+import Menu from "./Menu";
 import './App.css';
 
 function App() {
-  return (
-      <h1>Handbags & Purses</h1>
-  );
+    return (
+        <>
+            <h1>Handbags & Purses</h1>
+            <nav>
+                <Menu
+                    label="to the collection"
+                    logmessage="to the collection"
```

Comment:
Deze is eigenlijk overbodig. Als je in je menu functie in de console log label ipv logmassage mee geeft heb je zelfde resultaat en wordt je melding ook gelogd in de console. Minder code;)